src/DrupalCheck.php

```diff
@@ -33,9 +33,16 @@ public static function getConfigurableOptions(): OptionsResolver
   {
       $resolver = new OptionsResolver();
       $resolver->setDefaults([
-        'drupal_root' => null,
+        'drupal_root' => NULL,
+        'deprecations' => NULL,
+        'analysis' => NULL,
+        'php8' => NULL,
       ]);
       $resolver->addAllowedTypes('drupal_root', ['string', 'null']);
+      $resolver->addAllowedTypes('deprecations', ['boolean', 'null']);
```

Comment:
If we set the value as null this is a breaking change as `--deprecations` is added in the previous library versions. I would prefer to keep it as it is to:

- Don't force to have a major version.
- Check deprecated warnigns. Even though it is not the only feature drupal-check has, it is the most used one.

```suggestion
      $resolver->addAllowedTypes('deprecations', ['boolean', true]);
```